crates/live_kit_client/Cargo.toml

```diff
@@ -28,28 +28,23 @@ test-support = [
 
 [dependencies]
 anyhow.workspace = true
-async-broadcast = "0.7"
 async-trait = { workspace = true, optional = true }
-collections = { workspace = true, optional = true }
+cpal = { git = "https://github.com/zed-industries/cpal", rev = "fd8bc2fd39f1f5fdee5a0690656caff9a26d9d50" }
```

Comment:
As mentioned in https://github.com/zed-industries/zed/pull/13343#issuecomment-2294797663, the cpal dependency should be patched at the workspace level instead to avoid duplicate dependencies using the same version in the lockfile.

Reply:
https://github.com/zed-industries/zed/pull/13343/commits/eaac4a5731be2fda7faec4a2a059150ccbc91f42